### DISCUSSION TITLE
FR: Support Order/Sorting for FilePicker

### DISCUSSION THREAD

**Discussion by Dokdoy** [charmbracelet/huh#555] [Ideas]
e.g. Sort by Name, Size, Modified Time, etc.
So allow to separate folder and file, and sort by name, size, modified time, etc.

**Comment by bashbunni** [comment]
Hey @Dokdoy, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.
